htdocs/core/lib/ajax.lib.php

```diff
@@ -68,6 +68,7 @@ function ajax_autocompleter($selected, $htmlname, $url, $urloption = '', $minLen
 					$("input#search_'.$htmlname.'").keydown(function(e) {
 						if (e.keyCode != 9)		/* If not "Tab" key */
 						{
+							if (e.keyCode == 13) {return false;} /*disable "ENTER" key useful for barcode readers*/
```

Comment:
This seems ok.
However, i don't understand why changing parameter 0 to 1 into ajax_autocompleter
The goal of this parameter is to have field automatically filled when in the database, there is only 1 product. (in such a case, having a combo is useless).
So i merged manually this change only. Please check if it's ok.

Reply:
Hi eldy,
If you don't change the parameter 0 to 1 into ajax_autocompleter the input is slower if the product is unique (by hand as with a barcode reader). Indeed, the input line remains filled with the barcode and ajax displays below the product to be selected.  You have to click with the mouse or tap the down arrow on the keyboard to replace the barcode with the product code.
If the parameter is at 1, as soon as the barcode is complete, it is replaced on the same line by the product code, no further action is necessary. The speed is better.
When you do a lot during the day this small difference in ergonomics can be felt.